.github/workflows/linux.yml

```diff
@@ -501,15 +501,28 @@ jobs:
   #   image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
   #   volumes:
   #     - /mount:/mount
-  TensorFlow_Models_Tests:
+  TensorFlow_Models_Tests_Precommit:
     name: TensorFlow Models tests
     if: fromJSON(needs.smart_ci.outputs.affected_components).TF_FE.test ||
         fromJSON(needs.smart_ci.outputs.affected_components).TFL_FE.test
     needs: [ Build, Smart_CI, Openvino_tokenizers ]
     uses: ./.github/workflows/job_tensorflow_models_tests.yml
     with:
-      runner: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores' }}
-      event: ${{ github.event_name }}
+      runner: 'ubuntu-20.04-8-cores'
+      scope: 'precommit'
+
+  TensorFlow_Models_Tests_Nightly:
```

Comment:
why do we need a dedicated job for it?

Reply:
We need to separate nightly and precommit models scope. We can do it either by separating jobs or by adding one more parameter to `job_tensorflow_models_tests.yml`, which will define if it is precommit or nightly.